modules/gke-cluster-standard/variables.tf

```diff
@@ -38,11 +38,22 @@ variable "backup_configs" {
 variable "cluster_autoscaling" {
   description = "Enable and configure limits for Node Auto-Provisioning with Cluster Autoscaler."
   type = object({
+    autoscaling_profile = optional(string)
     auto_provisioning_defaults = optional(object({
       boot_disk_kms_key = optional(string)
+      disk_size         = optional(number)
+      disk_type         = optional(string)
```

Comment:
add validations to `autoscaling_profile` and `disk_type` to ensure only valid values are specified.

Reply:
Done in https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/commit/e2c030d8f7521f4f478a0058f7e9f40f69ca7a67 🙂